### DISCUSSION TITLE
What's our approach to conferences?

### DISCUSSION THREAD

**Discussion by Sealjay** [Green-Software-Foundation/oc#12] [General]
Do we have a set approach to conferences? I know we previously discussed KubeCon, or Open Source Summit. I realise this could directly fall under the Community working group @tmcclell and @aecurrie but I'm raising it as I often see other LF foundations, such as LF Energy represented at events like Open Source Summit 2022, or State of Open Con - stateofopencon.com/ - but very rarely us.

I worry about that because it's an underrated way to raise our profile in the community.

**Comment by tmcclell** [comment]
I love this idea, Chris. So, these would be paid events that GSF members could lead? Right now, conferences pay our speakers to come present. So it would need a budget of sorts? And we would need to have a targeted list of conferences?

**Reply by Sealjay** [reply]
That's right - I think we should target some of the big LF conferences, where our peers are, and make sure we have a presence to attract new contributors

**Reply by jawache** [reply]
I agree definitely think we need to be targeting conferences, @aecurrie said a while ago "there should be no major conference without a green software speaker" I've always loved that and now we have a large database of speakers at speakers.greensoftware.foundation and one of our OKRs is "Speakers give 100 green software talks at conferences in 2023"

1. The first thing we need is to **agree** on the list of major conferences.
2. The second thing is to figure out how to get someone to speak there.

For 1. I was going to raise in the community wg, but someone needs to take the lead and create the first stab at this.
For 2. I personally believe won't need to spend any money to get speaking slots. There is already a lot of interest from conferences, I've been asked at least 5 times so far this year - the demand is there for green speakers. 

At Microsoft our strategy was never to spend money earning speaking slots at conferences, those slots always have a tinge of "you bought you're way in, i'm not going to bother listening to you" and it gives some speakers imposter syndrome, they didn't earn their way on stage like the rest they just had a place bought for them. So they don't have a good rep. Instead the budget at MSFT mostly went on speaker training, I've had tons of pro speaker training, and writing support for CFPs. That's been my secret over the years.

I'd suggest we do the same. Get a list of conferences, find people who want to speak, give them lots of pro support writing CFPs, for the ones that get in give them pro speaker training so when they go on stage, they kill!

**Reply by Sealjay** [reply]
All great points, but I’m not talking about speaking - I’m talking about booths in the hallways etc. 

@aecurrie was at state of open con today for example; I understand we’ve been asked before to attend various events LF and other conferences - with a booth or a table - and I haven’t seen us show up yet.

I have seen other groups - LF Energy, FINOS, OCA, etc etc generate significant volunteer interest from these booths. Booths are static and present throughout an event, giving the chance to engage with content, for longer than a speaking slot does.

Speakers, I agree there’s a massive demand, and I’m not suggesting we change the approach the community WG is leading on.

edit: often for open source projects the booths are free, although rarely some do ask for a budget which is why I replied to @tmcclell in the affirmative above. The main resource constraint is staffing the booth with volunteers or paid for roles.

**Reply by jawache** [reply]
Ahh I completely misunderstood your point. You had me at free :) I agree this would be great. 

Is the first step the same? Identify the conferences we want a presence at?
Or are these conferences where there is a chance for a (free) booth a known/limited set - like the LF conferences for instance?

NOTE: I get nervous at the idea of sponsoring conferences generally, they are super expensive and in my experience rarely give you a good return. If there are free ways to get the same exposure I'd always vote for that :)

**Reply by greenhsu123** [reply]
we started identifying the conferences we want a presence at a while back, we have also been collecting who is speaking where @ the [Speaker's project](https://github.com/Green-Software-Foundation/speakers/issues/44)  (I say we, it's more Anita...)

**Reply by Sealjay** [reply]
Ok, that's a great help! Separating speaking from booths- would we have booths at conferences even if we aren't speaking? (I know we aim to have a speaker at every conference?)

**Comment by aecurrie** [comment]
At big confs, a booth usually needs sponsorship and Asim is right, not
cheap because it's the way the confs pay the huge bills. Chris though, I
agree it would be v nice to have. The commercials were screwy at OpenUK.
The tickets were way too cheap for that venue. We think Amanda got some
crazy deal, hence there were loads of non profit booths. Having said that,
I know DevOpsDays often turns a nice profit. Plus confs sometimes have free
space/tickets at the end that you can use last minute, the issue then is
getting folk to give up a day to man them. Sarah and I did it at a London
conf last year, but we couldn't find anyone else! It also requires banners
and stuff otherwise no one will see you.

Would there be a way to get a GSF guest at an existing booth? E.g. an
Avanade booth with some cross-promo?

I was amazed at the amount of green content at OpenUK (although the content
organiser is very pro-green). The tide has turned.

**Reply by Sealjay** [reply]
I think we might be able to work with the LF events team on getting a booth for GSF as another LF foundation - I'd be interested in knowing what the other foundations do.

And yes - that's a point. We'd have to buy pop-up and banners for those booths.

**Reply by jawache** [reply]
@seanmcilroy29 could you put all of us on a thread with LF events? Let's get introduced and we can ask them these questions.

**Reply by Sealjay** [reply]
Just following up on this one @jawache  @seanmcilroy29

**Reply by jawache** [reply]
@Sealjay might be better for this one if you just emailed sean directly? Adding people to an email thread might be easier for him

**Reply by Sealjay** [reply]
Done!
